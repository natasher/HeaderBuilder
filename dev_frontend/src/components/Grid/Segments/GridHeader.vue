<template>
  <header class="header">

    <dots />
    <view-with-resolution />
    <grid-status />
    <grid-tools />

  </header>
</template>

<script>
import GridStatus         from './GridStatus.vue'
import Dots               from './Dots.vue'
import ViewWithResolution from './ViewWithResolution.vue'
import GridTools          from './GridTools.vue'

export default {

  name: 'GridHeader',

  components: {
    Dots,
    GridStatus,
    ViewWithResolution,
    GridTools,
  },

}
</script>
