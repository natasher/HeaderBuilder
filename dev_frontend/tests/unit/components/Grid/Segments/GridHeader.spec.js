import { shallowMount } from '@vue/test-utils'
import GridHeader from '../../../../../src/components/Grid/Segments/GridHeader.vue'

describe('GridHeader.vue', () => {

  test('should match snapshot', () => {
    const wrapper = shallowMount( GridHeader )

    expect( wrapper.element ).toMatchSnapshot()
  })

})