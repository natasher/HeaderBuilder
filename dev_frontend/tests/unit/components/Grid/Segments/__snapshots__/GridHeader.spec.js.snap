// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridHeader.vue should match snapshot 1`] = `
<header
  class="header"
>
  <dots-stub />
   
  <viewwithresolution-stub />
   
  <gridstatus-stub />
   
  <gridtools-stub />
</header>
`;
